src/Microsoft.AspNet.Mvc.ModelBinding/Validation/DataAnnotationsModelValidator.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace Microsoft.AspNet.Mvc.ModelBinding.Validation
 {
-    public class DataAnnotationsModelValidator : IModelValidator, IClientModelValidator
+    public class DataAnnotationsModelValidator : IModelValidator
```

Comment:
Not sure I understand. Adapters are by definition server components... they translate server validator metadata into client validator metadata.

Then again, the key point of this change was to remove the interface from this type... everything else is just fallout from that. The purpose of this type is to take Data Annotations attributes and turn them into server validators.

Or was this comment about something else?